### PR TITLE
KEYCLOAK-18484 Documentation note about FAPI TLS considerations

### PR DESCRIPTION
**Pull request by mposolda**

### CODE REVIEW

**Comment by mposolda** [comment]
@andymunro @tnorimat Do you please have a chance to review this small PR?

**Review by tnorimat — DISMISSED** [review]
@mposolda LGTM.

File: securing_apps/topics/oidc/fapi-support.adoc [hunk 1]
```diff
@@ -7,6 +7,23 @@ and https://openid.net/specs/openid-financial-api-part-2-1_0.html[Financial-grad
...
+the cipher suites and TLS protocol versions used. To match these requirements, you can consider configure allowed ciphers. This can be done in the
+`KEYCLOAK_HOME/standalone/configuration/standalone-*.xml` file in the Elytron subsystem. For example this element can be added under `tls` -> `server-ssl-contexts`
+
+[source,xml]
+<server-ssl-context name="kcSSLContext" want-client-auth="true" protocols="TLSv1.2" \
+key-manager="kcKeyManager" trust-manager="kcTrustManager" \
+cipher-suite-filter="TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_256_GCM_SHA384" protocols="TLSv1.2" />
+
+The references to `kcKeyManager` and `kcTrustManager` refers to the corresponding Keystore and Truststore. See the documentation of Wildfly Elytron subsystem for more details and also
+other parts of {project_name} documentation like link:{installguide_link}#_network[Network Setup Section] or link:{adminguide_link}#_x509[X.509 Authentication Section].
```
Comment (andymunro) [inline securing_apps/topics/oidc/fapi-support.adoc]: Always replace like with "such as"

File: securing_apps/topics/oidc/fapi-support.adoc [hunk 2]
```diff
@@ -7,6 +7,23 @@ and https://openid.net/specs/openid-financial-api-part-2-1_0.html[Financial-grad
...
+
 To make sure that your clients are FAPI compliant, you can configure Client Policies in your realm as described in the link:{adminguide_link}#_client_policies[{adminguide_name}]
-and link them to the global client profiles for FAPI support, which are automatically available in each realm.
+and link them to the global client profiles for FAPI support, which are automatically available in each realm. You can use either `fapi-1-baseline` or `fapi-1-advanced` profile based on which FAPI
+profile you need your clients to conform with.
+
+==== TLS considerations
+
+As confidential information is being exchanged, all interactions shall be encrypted with TLS (HTTPS). Moreover, there are some requirements in the FAPI specification for
+the cipher suites and TLS protocol versions used. To match these requirements, you can consider configure allowed ciphers. This can be done in the
```
Comment (andymunro) [inline securing_apps/topics/oidc/fapi-support.adoc]: Please always follow "this" with a noun.

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good.  Just a couple of changes requested.

**Comment by mposolda** [comment]
@andymunro I've updated this pull request per your feedback. Thanks!

**Pull request merged**